.bumpversion.cfg

```diff
@@ -0,0 +1,7 @@
+[bumpversion]
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)|dev
+
+[bumpversion:file:setup.py]
```

Comment:
Do we need to have version in all those files?

Reply:
unfortunately yes, otherwise we would need to parse one file, IMO easier this way